src/v3/src/components/PhoneAuthenticator/PhoneAuthenticator.tsx

```diff
@@ -136,27 +137,42 @@ const PhoneAuthenticator: UISchemaElementComponent<UISchemaElementComponentWithV
 
   const renderCountrySelect = () => (
     <Box marginBottom={4}>
-      <NativeSelect
+      <InputLabel
+        id="countryLabel"
+        required={required}
```

Comment:
Ah, [FormControlLabel](https://mui.com/material-ui/api/form-control-label/#main-content) doesn't offer the `required` prop, so I had to add the `FormControl` wrapper which does. This is also the recommended approach for checkboxes, Radios, and Switches. 

While you can use a FormControl on a select or any other component, since it isnt required and since the InputLabel accepts the `required`, I decided against adding that wrapper.

Reply:
Ok sounds good!